src/scripts/rules.js

```diff
@@ -218,12 +218,52 @@ function requestMatcher(){
     for(let i = 0; i < parsedRulesList.length; i++){
         const rule = parsedRulesList[i]; 
         if(urlMatcher(request.httpRequestUrl, rule.urlParserIndexedRule) === true){
-            outputTextContent += JSON.stringify(rule, null, 2) + "\n\n";
             matchedRulesList.push(rule);
+            console.log("Matched rule: ", JSON.stringify(rule, null, 2));
         }
     }
+    let highestPriorityRules = getHighestPriorityRules(matchedRulesList);
+    for(let rule of highestPriorityRules){
+        outputTextContent += JSON.stringify(rule, null, 2) + "\n";
+    }
+    // console.log(outputTextContent);
```

Comment:
We probably don't need to commit this?

Reply:
Should I remove the `src/scripts` directory? The files aren't being used directly anymore, I have only kept them around for reference. Open to removing these files if you suggest so.

Reply:
Ah yes, removing them sounds good if they are now living elsewhere.